### DISCUSSION TITLE
how do i disable "typing behavior"

### DISCUSSION THREAD

**Discussion by sKopheK** [ohmyzsh/ohmyzsh#11514] [Help]
when a command is executed in zsh (e.g. debug from vscode), it's not pasted to the terminal at once, but the terminal is pretending to type it key by key which takes a lot of time instead of instant execution. where do i disable this behavior please?

**Comment by carlosala** [comment]
I do not understand your question, could you please clarify it with an example?

**Comment by sKopheK** [comment]
https://user-images.githubusercontent.com/5802917/220607540-7446bc43-899f-4e52-8c92-94da82f31ff2.mp4

**Reply by carlosala** [reply]
I'm not sure how is that being done actually. Could you try without `.zshrc`? That is, rename you `.zshrc` to `.zshrc.bak` and then it'll start a "clean" terminal also in `vscode`. Do you still have the same behaviour? To go back to your previous status, just re-rename your .zshrc 👌🏻

**Comment by sKopheK** [comment]
have renamed it and the behavior is gone but altogether with all neat features of zsh

**Reply by mcornella** [reply]
Might be due to `bracketed-paste-magic` or `url-quote-magic`. Can you try with the original .zshrc file but setting
```sh
DISABLE_MAGIC_FUNCTIONS=true
```
in your zshrc file, just before the line were %where OMZ is `source`d?

**Comment by sKopheK** [comment]
jave tried, didn't change the behavior
